src/connectedvmware/azext_connectedvmware/__init__.py

```diff
@@ -4,32 +4,28 @@
 # --------------------------------------------------------------------------------------------
 
 from azure.cli.core import AzCommandsLoader
-
-from azext_connectedvmware._help import helps  # pylint: disable=unused-import
+from azure.cli.core.commands import CliCommandType
+from azext_connectedvmware._client_factory import cf_connectedvmware
+from azext_connectedvmware._params import load_arguments
+from azext_connectedvmware.commands import load_command_table
```

Comment:
Actually you don't need to import module at top, disabling the linter rule would be better